src/FastExpressionCompiler/TestTools.cs

```diff
@@ -628,9 +629,9 @@ class CallerMemberNameAttribute : Attribute { }
 class CallerLineNumberAttribute : Attribute { }
 
 [AttributeUsage(AttributeTargets.Parameter, Inherited = false)]
-class CallerArgumentExpression : Attribute
+class CallerArgumentExpressionAttribute : Attribute
```

Comment:
There would be a conflict otherwise in internal files in some fwks, it would be conflicted between this and the real ones when it's used later in the code.

Reply:
Funny, that missed it.